### DISCUSSION TITLE
Achieving Focus on iframe Elements within Custom UI

### DISCUSSION THREAD

**Discussion by 8128** [PierfrancescoSoffritti/android-youtube-player#1113] [General]
Thank you for creating this library. I'm curious if there's a method to programmatically focus on an iframe element using physical buttons. I'm developing a TV app and have been using a custom UI. To comply with Google Play's requirements, every button displayed on the page must be accessible to the user, including those for ending recommendations and pausing recommendations. In my testing, it seems I'm unable to focus on those elements. I'm wondering if there's a solution for this. Thank you!

**Comment by PierfrancescoSoffritti** [comment]
Hi, personally I don't know the answer to your question, but I would expect yes. There should be a way to set the focus on the IFrame and be able to cycle the focus through the UI elements in the IFrame.

**Comment by PierfrancescoSoffritti** [comment]
ChatGPT suggests doing custom focus handling on the web side

```
window.addEventListener('keydown', function(event) {
    switch(event.key) {
        case "ArrowUp":
            // Move focus up
            break;
        case "ArrowDown":
            // Move focus down
            break;
        case "ArrowLeft":
            // Move focus left
            break;
        case "ArrowRight":
            // Move focus right
            break;
    }
});
```

You could try to play with this to verify if it works.
